src/main/java/frc/robot/superstructure/Superstructure.java

```diff
@@ -45,55 +55,135 @@ public Command setState(RobotState state) {
 	//@formatter:off
 	public Command idle() {
 		return new ParallelCommandGroup(
-				swerve.getCommandsBuilder().saveState(SwerveState.DEFAULT_DRIVE)
+				swerve.getCommandsBuilder().saveState(SwerveState.DEFAULT_DRIVE),
+				funnelStateHandler.setState(FunnelState.STOP)
+				//elevatorRoller.STOP
+				//intakeRoller.STOP
+				//flywheel.DEFAULT
+				//intakePivot.UP
+				//elevator.IDLE
 		);
 	}
 
 	public Command intake() {
 		return new ParallelCommandGroup(
+				new SequentialCommandGroup(
+						new ParallelCommandGroup(
+								//intakePivot.INTAKE
+								//intakeRoller.INTAKE
+								funnelStateHandler.setState(FunnelState.STOP)
+						).withTimeout(3), // .until(() -> isNoteInFunnel
```

Comment:
```suggestion
						).withTimeout(3), // .until(() -> isNoteInIntake 
```
i think it will be more understandable